index/scorch/scorch.go

```diff
@@ -303,6 +318,7 @@ func (s *Scorch) Reader() (index.IndexReader, error) {
 	rv := &Reader{root: s.root}
 	rv.root.AddRef()
 	s.rootLock.RUnlock()
+	atomic.AddUint64(&s.stats.termSearchersStarted, 1)
```

Comment:
termSearchersStarted/Finished is supposed to count TermFrequencyReaders, not IndexReaders